### DISCUSSION TITLE
What's new

### DISCUSSION THREAD

**Discussion by Nex-Otaku** [Nex-Otaku/php-rfc#20] [Show and tell]
Added RFC drafts based on ideas from blog [stitcher.io](https://stitcher.io/blog/php-reimagined) by Brent Roose.

Actually I just copy-pasted ready made suggestions, with basically no editing at all. I attached link to original post to each RFC draft to mention author.

I think all of these ideas definitely worthy to be discussed. 

Some of them I really want to be implemented.

**Comment by Nex-Otaku** [comment]
Added two more RFC ideas:
[Private properties by default](https://github.com/Nex-Otaku/php-rfc/issues/21)
[Private methods by default](https://github.com/Nex-Otaku/php-rfc/issues/22)

**Comment by Nex-Otaku** [comment]
Added "A letter to the PHP team" by Brent Roose for discussion. 

Please tell what you think about it: https://github.com/Nex-Otaku/php-rfc/discussions/24

**Reply by badmansan** [reply]
> 
> 
> Added "A letter to the PHP team" by Brent Roose for discussion.
> 
> Please tell what you think about it: #24

It looks like nothing has changed in 1.5 years...

**Comment by Nex-Otaku** [comment]
RFC [Final on variables](https://github.com/Nex-Otaku/php-rfc/issues/13) was closed, as it was implemented in PHP 8.1 with "readonly properties"
